Doc/library/sqlite3.rst

```diff
@@ -1868,12 +1868,16 @@ How to use placeholders to bind values in SQL queries
 
 SQL operations usually need to use values from Python variables. However,
 beware of using Python's string operations to assemble queries, as they
-are vulnerable to `SQL injection attacks`_ (see the `xkcd webcomic
-<https://xkcd.com/327/>`_ for a humorous example of what can go wrong)::
-
-   # Never do this -- insecure!
-   symbol = 'RHAT'
-   cur.execute("SELECT * FROM stocks WHERE symbol = '%s'" % symbol)
+are vulnerable to `SQL injection attacks`_. For example::
+
+   >>> # Never do this -- insecure!
+   >>> # An attacker can select all stocks with a crafted input.
```

Comment:
Perhaps make this a part of the text above and not the doctest?

Reply:
IMO, we should explicitly mention in the introductory text that the following is an example of what _not_ to do as it is how such an attack could be conducted, but IMO the warning comments also have a place in the example, to make fully sure that if readers just jump to the example, or they view/copy it elsewhere, it is unambiguously an example of something _not_ to do.

Reply:
Yeah, we definitely need to keep the warning as a part of the doctest, specifically with C&P in mind.